airflow/providers/docker/decorators/docker.py

```diff
@@ -79,7 +79,7 @@ def __init__(
             command=command, retrieve_output=True, retrieve_output_path="/tmp/script.out", **kwargs
         )
 
-    def execute(self, context: Dict):
+    def execute(self, context: Any):
```

Comment:
```suggestion
    def execute(self, context: Context):
```

I think?

Reply:
Would this typing make this version of the Docker provider have a lower bound of a newer Airflow version since `Context` comes from `airflow.utils.context`? I may have incorrectly assumed that here when I chose `Any`.

Reply:
Oh arg. Good spot, yes. Ignore this, or maybe just leave it as a Dict then? (Cos a Context isa Dict)

Reply:
Unfortunately MyPy complains about the typing difference here between `_DockerDecoratedOperator` and `DecoratedOperator`. `DecoratedOperator` has `context: Context` to match `BaseOperator`. Using `Union[Context, Dict]` on `DecoratedOperator` works of course but seems redundant.